### ISSUE TITLE
Documentation link on README when creating a repo is broken

### ISSUE THREAD

**Issue by cubiclesoft** [gogs/gogs#5618]
**Describe the bug**
When creating a new repo in Gogs, there is a little help icon next to the Readme field.  Click that icon goes to a broken page that a couple of months ago contained content but now contains only a few Chinese characters:

https://github.com/gogs/go-gogs-client/wiki/Repositories#litte-notes-on-readme-template

Google Translate says, "Xinghan Autonomous System".  The commit that removed all of the content that was on the Wiki page has a commit message of "clean".

**To Reproduce**
1. Start creating a new repo.
2. Click on the help icon next to the `Readme` field.

**Expected behavior**
A page of documentation.

**Additional context**
- Gogs version (or commit ref): 0.11.86.0130
- Git version: N/A
- Operating system: N/A
- Database (replace `[ ]` with `[x]`):  N/A
  - [x] SQLite
- Can you reproduce the bug at https://try.gogs.io:
  - [x] No: Too lazy to create an account.  Unless it's somehow been fixed already, it's probably the same.
- Log gist (usually found in `log/gogs.log`):  N/A

**Comment by DanLanglois** [comment]
'* Can you reproduce the bug at https://try.gogs.io:'

Yes, it's there.

![image](https://user-images.githubusercontent.com/1574379/52784073-c38a4080-3008-11e9-9b04-b52421858a2f.png)

**Comment by DanLanglois** [comment]
At https://github.com/gogs/go-gogs-client/wiki/Repositories#litte-notes-on-readme-template:

My change to the linked wiki page looks like this now:

![image](https://user-images.githubusercontent.com/1574379/52788335-72347e00-3015-11e9-8dff-e590517d7c35.png)

**Comment by Unknwon** [comment]
Thanks for the feedback!

However, when I clicked on the "?" icon, my Chrome correctly redirects me to the link "https://github.com/gogs/go-gogs-client/wiki/Repositories#litte-notes-on-readme-template". GitHub automatically replaces old "gogits" to "gogs" for me.

**Comment by cubiclesoft** [comment]
The problem is that the linked Wiki page did not contain any content.  Looks like @DanLanglois put some of the content back up there.  Look at the history on that page.

**Comment by Unknwon** [comment]
Yeah, you're right, the link should be updated in the main repository.
